grpc-native/src/main/java/org/ballerinalang/net/grpc/protobuf/cmd/GrpcCmd.java

```diff
@@ -74,15 +74,13 @@
 
     private static final PrintStream outStream = System.out;
     private static final String PROTO_EXTENSION = "proto";
-    private static final String WHITESPACE_CHARACTOR = " ";
-    private static final String WHITESPACE_REPLACEMENT = "\\ ";
 
     private CommandLine parentCmdParser;
 
     @CommandLine.Option(names = {"-h", "--help"}, hidden = true)
     private boolean helpFlag;
 
-    @CommandLine.Option(names = {"--input"}, description = "Input .proto file")
+    @CommandLine.Option(names = {"--input"}, description = "Input .proto file(s)")
```

Comment:
Here, the description `files(s)` seems like it allows to add a list of files. But here we need a directory as input. Shall we add a description, something like:

> Input proto file or a directory containing multiple proto files

Reply:
Updated in 6486d05